frontend/src/app/Settings/Facility/Components/OrderingProvider.tsx

```diff
@@ -105,66 +105,94 @@ const OrderingProvider: React.FC<Props> = ({
           }
           name="NPI"
           required={isRequired}
-          value={provider.NPI || ""}
-          onChange={onChange}
-          onBlur={() => {
-            validateField("orderingProvider.NPI");
-          }}
+          value={formCurrentValues.orderingProvider.NPI ?? undefined}
+          registrationProps={register("orderingProvider.NPI", {
+            required: isRequired
+              ? "NPI should be a 10-digit numerical value (##########)"
```

Comment:
Could we pull these error messages into a shared constants file? Just in case we need to change the error status slightly in the future or if we do any checks against the error messages themselves, don't want to have potential typo / string drift problems. I'll leave a note to other instances as well just so you have an easy reference.

Reply:
Thank you, Bob! Will do!